packages/flutter_bloc/pubspec.yaml

```diff
@@ -11,7 +11,7 @@ environment:
   sdk: ">=2.12.0 <4.0.0"
 
 dependencies:
-  bloc: ^8.1.1
+  bloc: ^8.1.2
```

Comment:
This isn’t necessary because ^8.1.1 includes 8.1.2. Closing for now but if you feel this is necessary feel free to file an issue, thanks!